src/pages/developers/tutorials.js

```diff
@@ -259,9 +278,9 @@ const TutorialsPage = ({ data, pageContext }) => {
     }
 
     // If no tags are active, show all tutorials, otherwise filter by active tag
-    let filteredTutorials = allTutorials
+    let filteredTutorials = [].concat(externalTutorials, allTutorials)
     if (activeTagNames.length > 0) {
-      filteredTutorials = allTutorials.filter((tutorial) => {
+      filteredTutorials = state.filteredTutorials.filter((tutorial) => {
```

Comment:
Currently, all tutorials are filtered by date in descending order (this is done by GraphQL for free).

With this the external tutorials will always be displayed first and then the ordered internal tutorials. We're probably going to want to combine external and internal tutorials and order them by date descending (are we best doing this all with GraphQL @wackerow ?).

![Screenshot 2021-08-16 at 23 35 24](https://user-images.githubusercontent.com/62268199/129637533-e5de94d5-4d07-478a-a4f6-2613657eaf84.png)

Reply:
They are one array in state, so it would be possible to sort them by date.